dotnet/src/Experimental/Agents.UnitTests/Integration/ThreadHarness.cs

```diff
@@ -68,4 +68,32 @@ public async Task VerifyThreadLifecycleAsync()
 
         await Assert.ThrowsAsync<HttpOperationException>(() => context.GetThreadModelAsync(thread.Id)).ConfigureAwait(true);
     }
+
+    /// <summary>
+    /// Verify retrieval of thread messages
+    /// </summary>
+    [Fact(Skip = SkipReason)]
```

Comment:
Should this new be skipped? If so, a more descriptive reason would be good.

Reply:
Everything in this folder is disabled by default since they aren't unit-tests, they really dev-harnesses posing as integration tests.  Each file has a descriptor:

![image](https://github.com/microsoft/semantic-kernel/assets/66376200/ad88202a-c05a-4640-8b17-965a1d7d3c24)

Github review tool hides this unless you dig!